doc/workspace_symbol_params.md

```diff
@@ -0,0 +1,56 @@
+# WorkspaceSymbolParams
+
+## Short introduction
+
+This feature allows the LSP server to provide results for more flexible
+search conditions like:
+  -  A Full_Text match searches the pattern exactly in the contents.
```

Comment:
Full_Text is the _default_ choice, right? This mean this PR breaks compatibility.
* Before PR the pattern `X` matches `XA` symbol, but doesn't match `AX` symbol.
* After PR the pattern `X` matches both `XA` and `AX` symbols.

I would prefer to keep old behavior "by default" (in absence of our custom fields).